main/2021/CG-08-03.md

```diff
@@ -27,7 +27,14 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on [Branch Hinting](https://github.com/WebAssembly/branch-hinting) [20 min].
         1. Poll for phase 3
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
+        1. Poll for supporting [UTF-16/Latin1](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
+        1. Poll for supporting `DOMString` in an IT `string` type
```

Comment:
This poll is essentially the list-of-USV poll with the for/against positions reversed, and the associated editorialising is not appropriate.

Reply:
I do think this is very appropriate, given that the group is used to vote favorably or not at all (the direction a question is asked is relevant - or can we merge this to an "or" question, not a "for" or "against"?), and since list-of-USVs implies a fundamental directional change of the entire standard that may forever bias it towards Non-Web use cases and very particular languages, that are not JavaScript, I think it makes a lot of sense to give context on what we are actually voting on.

Reply:
Reordered the higher level questions to a discussion, and made the vote an "or" :)